packages/rocketchat-api/server/v1/rooms.js

```diff
@@ -0,0 +1,93 @@
+RocketChat.API.v1.addRoute('rooms.get', { authRequired: true }, {
+	get() {
+		const { updatedSince } = this.queryParams;
+
+		let updatedSinceDate;
+		if (updatedSince) {
+			if (isNaN(Date.parse(updatedSince))) {
+				throw new Meteor.Error('error-updatedSince-param-invalid', 'The "updatedSince" query parameter must be a valid date.');
+			} else {
+				updatedSinceDate = new Date(updatedSince);
+			}
+		}
+
+		let result;
+		Meteor.runAsUser(this.userId, () => result = Meteor.call('rooms/get', updatedSinceDate));
+
+		if (Array.isArray(result)) {
+			result = {
+				update: result,
```

Comment:
shoud this return a field `results` or `rooms` instead of `update` and `remove`?